kitchen.recipes.yml

```diff
@@ -18,14 +19,15 @@ suites:
       # runs the recipe to test, along with the recipes it depends on
       - recipe[aws-parallelcluster::test_dummy]
       - recipe[aws-parallelcluster::example_recipe]
+      - recipe[aws-parallelcluster-install::sudoers]
       #- recipe[aws-parallelcluster-install::base]
-    verifier:
-      controls:
+    #verifier:
+      #controls:
```

Comment:
Please uncomment: this is the part where you pick specific controls to run.

Reply:
Uncommented